src/Label/Label.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import {Meta, Story} from '@storybook/react'
+import {ComponentProps} from '../utils/types'
+import Label from './Label'
+
+export default {
+  title: 'Components/Label',
+  component: Label,
+  args: {
+    variant: 'default',
+    size: 'small',
+  },
+  argTypes: {
```

Comment:
Can we apply `args` and `argTypes` directly to the playground story?

```
Playground.args = {...}
Playground.argTypes = {...}
```

The way this is written, I think the Default story inherits these args.

Reply:
Yes, thanks for the reminder!